src/components/pages/observation/list/views/stats/table-uploaders.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box, Button, Skeleton, Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+import ExternalBlueLink from "@components/@core/blue-link/external";
+import BoxHeading from "@components/@core/layout/box-heading";
+import useTranslation from "@hooks/use-translation";
+import React from "react";
+import { getUserImage } from "@utils/media";
+
+import { Avatar, HStack } from "@chakra-ui/react";
+
+import { stringify } from "querystring";
+
+export default function UploadersTable({ data, title, loadMoreUniqueSpecies, filter }) {
+  const { t } = useTranslation();
+
+  const queryParams = { ...filter };
+  if (filter.user) {
+    delete queryParams["user"];
+  }
```

Comment:
please avoid mutations here Instead of this you can just destructure by excluding single value like below

```jsx
const { user: _user, ...queryParams } = filter;
const address = stringify({ ...queryParams, view: "list" });
```

or even easier single liner
```jsx
const address = stringify({
  ...filter,
  view: "list",
  user: undefined,
});

```

Reply:
used the first approach provided